File: Sesion 6/Marco teorico/3CÓMO SE DECLARAN LOS ARREGLOS DE OBJETOS/3.java
class Persona {
    private String nombre;
    private int edad;

    public Persona(String nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
    }

    public void mostrarInfo() {
        System.out.println("Nombre: " + nombre + ", Edad: " + edad);
    }
}

public class Main {
    public static void main(String[] args) {
        Persona[] personas = {
            new Persona("Juan", 25),
            new Persona("María", 30),
            new Persona("Carlos", 40)
        };

        for (Persona persona : personas) {
            persona.mostrarInfo();
        }
    }
}

File: Sesion 6/Marco teorico/3CÓMO SE DECLARAN LOS ARREGLOS DE OBJETOS/3.py
class Persona:
    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.edad = edad

    def mostrar_info(self):
        print("Nombre:", self.nombre, ", Edad:", self.edad)



personas = [
    Persona("Juan", 25),
    Persona("María", 30),
    Persona("Carlos", 40)
]

for persona in personas:
    persona.mostrar_info()